particles/Renderer.cpp

```diff
@@ -195,7 +195,7 @@ void Renderer::init(score::gfx::RenderList& renderer)
             sbuf.d[index].spd = s;
         }
     }
-    )_").arg(instances);
+    )_").arg(maxparticles);
```

Comment:
instead of maxparticles here, shouldn't it be particlesNumber ? otherwise it makes computation for unneeded particles, no ?

Reply:
If I put particlesNumber here, it will not initialize the remaining speeds and offsets so the particles added afterward don't have any movement.